src/main/asciidoc/shareddata.adoc

```diff
@@ -92,6 +95,8 @@ is obtained.
 
 To obtain a lock use {@link io.vertx.core.shareddata.SharedData#getLock(java.lang.String,io.vertx.core.Handler)}.
 
+To obtain a local lock, even if Vert.x is clustered, use {@link io.vertx.core.shareddata.SharedData#getLocalLock(java.lang.String,io.vertx.core.Handler)}.
```

Comment:
Can you rework this part of the doc to look like the map one? An `IMPORTANT` callout indicating that in clustered mode, the lock relies on distributed data structures ...etc and if you don't need this, invoke `getLocalLock` ?